src/request/export/Export.css

```diff
@@ -43,7 +43,7 @@ ul.nq-list li > .nq-icon {
 }
 
 .page#recovery-words .recovery-words {
-    height: 39rem;
+    height: 37rem;
```

Comment:
Does this have anything to do with the skip buttons? Otherwise I disagree with it.